README.md

```diff
@@ -102,6 +102,39 @@ Array<{
 };
 ```
 
+### `/api/getPoolInfo`
+
+#### Input
+
+```js
+{
+  poolMetaDataHashes: Array<string> //hash ids of pool meta data.
+};
+```
+
+#### Output
+
+```js
+{
+  [poolMetaDataHash: string]: {| info: RemotePoolInfo,
```

Comment:
confused about what is returned if the pool isn't found. It used to be null, right? Now is it an empty info array and empty history?

Reply:
it'll be an empty object, `{}`

Reply:
Do you mean like this?
```
{
    'df1750df9b2df285fcfb50f4740657a18ee3af42727d410c37b86207': {},
    'aadf1750df9b2df285fcfb50f4740657a18ee3af42727d410c37b862aa': {},
}
```

If so, I would prefer not returning an empty object if possible because JS isn't really good at dealing with empty objects

Reply:
it is now something like:
```js
{
  "a0b42a87829a0260f36d7f500b2142d0fd0bbe127c77110870db91340f4fdca0": {
    "info": {
      "pledge_address": null
    },
    "history": []
  }
}
```